packages/i18nify-js/src/modules/currency/__tests__/convertToMinorUnit.test.ts

```diff
@@ -7,12 +7,12 @@ describe('currency - convertToMinorUnit', () => {
     currency: CurrencyCodeType;
     expectedResult: number;
   }[] = [
-    { amount: 1, currency: 'USD', expectedResult: 100 },
-    { amount: 1, currency: 'GBP', expectedResult: 100 },
+    { amount: 1, currency: 'USD' as CurrencyCodeType, expectedResult: 100 },
```

Comment:
Why do we have to type cast here ?

Reply:
Not required. Added this coz there was an issue with local lint extension issue. Removing this.